docs/content/get-started/set-up.md

```diff
@@ -140,25 +140,38 @@ If you need help using the GDK, come and talk to us about the software and the d
 <br/>
 
 
-## Clone the repos
+## Get the GDK and Starter Project Source Code
+
+### Zip Download
+
+If you are unfamiliar with Git, you can download a Zip file of the GDK and its Starter Project [here](https://github.com/spatialos/gdk-for-unity-fps-starter-project/releases/tag/0.1.3). 
+
+Once you have done that, you can skip the rest of this page and move on to [Open the FPS Starter Project]({{urlRoot}}/content/get-started/open-project.md).
+
+<%(Callout type="alert" message="
+If you can, please use Git to get the GDK and its starter project, as version control will make it easier for you to get updates in the future.")%>
+
+### Cloning the Repos using Git
 
 You need to clone two repositories; the FPS Starter Project and the GDK for Unity.
 
-### Clone the FPS Starter Project repository
+#### Clone the FPS Starter Project repository
 
 Clone the FPS starter project using one of the following commands:
 
-
 |     |     |
 | --- | --- |
-| SSH | `git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git` |
 | HTTPS | `git clone https://github.com/spatialos/gdk-for-unity-fps-starter-project.git` |
+| SSH | `git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git` |
+
+<%(Callout type="warning" message="
+We recommend using SSH - if you're not using SSH to connect to Github, follow [this handy guide](https://help.github.com/articles/connecting-to-github-with-ssh/) to set it up.")%>
```

Comment:
I don't think we recommend something either way - I'd rephrase this to be more like

```
You can only clone via SSH if you have already [set up SSH keys](https://help.github.com/articles/connecting-to-github-with-ssh/) with your GitHub account.
```